Configuration/DataProcessing/python/Reco.py

```diff
@@ -157,8 +157,12 @@ def visualizationProcessing(self, globalTag, **args):
 
         eiStep=''
 
-        options.step += 'RAW2DIGI,L1Reco,RECO'+eiStep+',ENDJOB'
-
+        if 'beamSplashRun' in args:
+          recoDQMoptions = ":localreco+hcalOnlyGlobalRecoSequence+caloTowersRec" if args['beamSplashRun'] else ""
+          options.step += 'RAW2DIGI,L1Reco,RECO'+recoDQMoptions+eiStep+',ENDJOB'
```

Comment:
`eiStep` is still there: just forgotten?

Reply:
eiStep was here before my commit, eiStep exist in all other *Processing functions